Makefile

```diff
@@ -93,129 +92,49 @@ release: certs docker-release
 	@docker build -f scripts/dockerfiles/Dockerfile.release -t "amazon/amazon-ecs-agent:latest" .
 	@echo "Built Docker image \"amazon/amazon-ecs-agent:latest\""
 
-gogenerate:
-	go generate -x $(shell go list ./agent/...)
-	$(MAKE) goimports
-
 # We need to bundle certificates with our scratch-based container
 certs: misc/certs/ca-certificates.crt
 misc/certs/ca-certificates.crt:
 	docker build -t "amazon/amazon-ecs-agent-cert-source:make" misc/certs/
 	docker run "amazon/amazon-ecs-agent-cert-source:make" cat /etc/ssl/certs/ca-certificates.crt > misc/certs/ca-certificates.crt
 
-ifeq (${BUILD_PLATFORM},aarch64)
-test::
-	go test -tags unit -timeout=30s -v -cover $(shell go list ./agent/... | grep -v /vendor/)
-else
-test::
-	go test -race -tags unit -timeout=30s -v -cover $(shell go list ./agent/... | grep -v /vendor/)
-endif
-
-ifeq (${BUILD_PLATFORM},aarch64)
-test-silent::
-	go test -tags unit -timeout=30s -cover $(shell go list ./agent/... | grep -v /vendor/)
-else
-test-silent::
-	go test -race -tags unit -timeout=30s -cover $(shell go list ./agent/... | grep -v /vendor/)
-endif
-
-benchmark-test:
-	go test -run=XX -bench=. $(shell go list ./agent/... | grep -v /vendor/)
+gogenerate:
+	go generate -x ./agent/...
```

Comment:
I can't find documentation about `$(shell go list ./agent/...)` vs `./agent/...`
Can you explain this change?  is there an update to the generate that means we no longer need to list, or was this just un-necessary in the first place?

Reply:
I mentioned it in another PR (I don't think its worth putting references to old code in current code). Basically,

`go test ./my-package/...` used to also do things to ./my-package/vendor/. They eventually made it so that go test / list didn't expand to include the vendored directory by default.

We were using `go list ./agent/... | grep -v vendor` to exclude manually, but thats no longer needed.